frontend/layouts/default.tsx

```diff
@@ -1,13 +1,32 @@
+import { Header } from '@/components/shared/header'
 import { Link } from '@nextui-org/link'
+import NextHead from 'next/head'
+import Script from 'next/script'
 
 export default function DefaultLayout({
   children,
 }: {
   children: React.ReactNode
 }) {
   return (
-    <div className="relative flex flex-col h-screen">
-      <main className="container mx-auto max-w-7xl px-6 flex-grow">
+    <div className="relative flex flex-col h-screen justify-center items-center">
+      <NextHead>
+        <title>sea-bit</title>
+        <meta key="title" content={'sea-bit'} property="og:title" />
+        <meta content={'free diving booking'} property="og:description" />
+        <meta content={'free diving booking'} name="description" />
+        <meta
+          key="viewport"
+          content="viewport-fit=cover, width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+          name="viewport"
+        />
+        <link href="/favicon.ico" rel="icon" />
+        <link href="./output.css" rel="stylesheet" />
+      </NextHead>
+      {/* <Script src="https://cdn.tailwindcss.com" /> */}
+      <Header />
+      {/* TODO: change padding for responsiveness */}
+      <main className="container mx-auto px-6 px-6 lg:px-20 xl:px-36 2xl:px-44 flex-grow">
```

Comment:
The classes
`mx-auto px-6 px-6 lg:px-20 xl:px-36`
are also used inside the Header component to sync the padding of the main website body and header. We may or may not create a custom class for this